Tests/iaas/flavor-naming/flavor-name-check.py

```diff
@@ -700,9 +713,14 @@ def main(argv):
             continue
         scs += 1
         namecheck = outname(*ret)
-        if completecheck and name in scsMandatory:
-            scsMandatory.remove(name)
-
+        if completecheck:
+            if name in scsMandatory:
+                scsMandatory.remove(name)
+            elif accept_old_mand:
+                newnm = old_to_new(name)
+                # print(f"DEBUG: {name} -> {newnm} in list: {newnm in scsMandatory}")
```

Comment:
not really a big fan of debug code that is left - even with commented out. but that is personal taste.